doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt

```diff
@@ -27,3 +27,9 @@ set(project_SOURCES main.cpp pclviewer.cpp)
 add_executable(${PROJECT_NAME} ${project_SOURCES})
 
 target_link_libraries(${PROJECT_NAME} ${PCL_LIBRARIES} Qt5::Widgets)
+
+#TODO: Update when CMAKE 3.10 is available
```

Comment:
Isn't it the minimum required version already?

Reply:
Yes, but I wrote this as part of the vtk 9 branch. Just havent got around to update it after the bump to 3.10 :-)

Reply:
Yes. And yes 😄